### ISSUE TITLE
Element is not clickable at location

### ISSUE THREAD

**Issue by razvanilin** [InstaPy/InstaPy#42]
I get this error and the script crashes after about 5-10 likes:

```
selenium.common.exceptions.WebDriverException: Message: unknown error: Element <span class="_soakw coreSprite...HeartOpen">Like</span> is not clickable at point (445, 463). Other element would receive the click: <div class="_mghjk">...</div>
  (Session info: chrome=57.0.2987.133)
  (Driver info: chromedriver=2.29.461571 (8a88bbe0775e2a23afda0ceaf2ef7ee74e822cc5),platform=Linux 4.4.0-72-generic x86_64)
```

**Comment by timgrossmann** [comment]
@razvanilin I also got this error a few times already... I think this is due to the footer bar in the Instagram webpage :🙁
We'll see if we can fix this.

Thank you for reporting!

**Comment by razvanilin** [comment]
At first I thought it was because I was using a business account. I then reverted back to personal account and it still happens. So business accounts are not causing this.

**Comment by timgrossmann** [comment]
@razvanilin Thank you, I really think it's the footer bar in Instagram, since Selenium just moves to the element and clicks the mentioned element.

**Comment by caltv** [comment]
yes I'm seeing the same

selenium.common.exceptions.WebDriverException: Message: unknown error: Element <span class="_soakw coreSpriteHeartOpen">...</span> is not clickable at point (684, 678). Other element would receive the click: <div class="_mghjk">...</div>

**Comment by caltv** [comment]
Update: it seems if you manually click that footer at the beginning to dismiss it, the script continues successfully.

**Comment by razvanilin** [comment]
I tried that in the beginning but it didn't work for me. What finally worked is to switch my account back to business account.

**Comment by eduardocalazansjr** [comment]
@caltv what footer is that? i don't have it.
could you post a screenshot of the footer and tell me what resolution are you using?

**Comment by caltv** [comment]
<img width="882" alt="screen shot 2017-04-07 at 15 03 21" src="https://cloud.githubusercontent.com/assets/219204/24803510/604d53de-1ba3-11e7-934f-6ee997d6f95b.png">

**Comment by caltv** [comment]
I'm on a retina 2880 x 1800 res

**Comment by timgrossmann** [comment]
Fixed by #47
